.circleci/config.yml

```
@@ -0,0 +1,64 @@
+version: 2
+
+aliases:
+  - &restore-cache
+    restore_cache:
+        key: dependency-cache-{{ checksum "package.json" }}
```

```suggestion
        key: dependency-cache-{{ checksum "package-lock.json" }}
```
If you're using npm ci it makes more sense to cache by the lock file